wtxmgr/tx.go

```diff
@@ -260,50 +208,43 @@ func (s *Store) moveMinedTx(ns walletdb.ReadWriteBucket, rec *TxRecord, recKey,
 		if err != nil {
 			return err
 		}
+
 		cred.outPoint.Index = index
 		cred.amount = amount
 		cred.change = change
 
-		err = putUnspentCredit(ns, &cred)
-		if err != nil {
+		if err := putUnspentCredit(ns, &cred); err != nil {
 			return err
 		}
 		err = putUnspent(ns, &cred.outPoint, &block.Block)
 		if err != nil {
 			return err
 		}
 
-		// reposition cursor before deleting, since the above puts have
-		// invalidated the cursor.
-		it.reposition(&rec.Hash, index)
```

Comment:
why is this not needed anymore?

Reply:
We're no longer deleting the credits within the iterator, so I believe there should be no need for this.